### PR TITLE
SDKS-2097 Fix Google dependency vulnerabilities in the SDK

### PR DESCRIPTION
**Pull request by witrisna**
# JIRA Ticket
[SDKS-2097](https://bugster.forgerock.org/jira/browse/SDKS-2097)

# Description
Even upgrading to the latest dependency libraries from Google, Google libraries still depend on order version of  'com.google.android.gms:play-services-basement', to resolve the vulnerabilities issue, explicitly include the latest version of  'com.google.android.gms:play-services-basement:18.1.0' is required

The following components have been tested with the upgraded Google Libraries:

- Captcha Node
- Capture Device Location
- Social Login with Google
- Centralize Login

# Definition of Done Checklist:

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Ensure backward compatibility.
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

**Review by spetrov — APPROVED** [review]
LGTM 👍🏻

File: forgerock-auth-ui/build.gradle [hunk 1]
```diff
@@ -42,9 +42,6 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
-    compileOnly 'com.google.android.gms:play-services-fido:18.1.0'
```
Comment (rodrigoareis) [inline forgerock-auth-ui/build.gradle:45]: Update copyright

File: forgerock-auth/src/test/java/org/forgerock/android/auth/SocialLoginTest.java [hunk 2]
```diff
@@ -172,7 +172,7 @@ public void onException(Exception e) {
             }
         });
 
-        SignInCredential signInCredential = new SignInCredential("1234", "", "", "", null, "", "dummy_id_token");
+        SignInCredential signInCredential = new SignInCredential("1234", "", "", "", null, "", "dummy_id_token", "");
```
Comment (rodrigoareis) [inline forgerock-auth/src/test/java/org/forgerock/android/auth/SocialLoginTest.java:175]: Update copyright

File: samples/authenticator/build.gradle [hunk 3]
```diff
@@ -51,7 +51,7 @@ dependencies {
     implementation 'com.squareup.picasso:picasso:2.5.2'
 
     // FCM Notifications
-    implementation "com.google.firebase:firebase-messaging:20.2.0"
+    implementation "com.google.firebase:firebase-messaging:23.0.8"
```
Comment (rodrigoareis) [inline samples/authenticator/build.gradle:54]: Update copyright

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Changes looks good to me. Just missing few header updates

**Review by spetrov — APPROVED** [review]

**Pull request merged**